source/collections.txt

```diff
@@ -70,33 +94,35 @@ Drop a Collection
 
 .. _collection-details:
 
-View Collection Details
------------------------
+Collection Details
+------------------
 
-The Collection Detail view shows detailed information for a
-collection, including the documents the collection contains. To view
+The Collection Detail screen shows detailed information for a
+collection, including the documents the collection contains. To see
 collection details, either:
 
 - Click a :guilabel:`Collection Name` in the main
-  :guilabel:`Collections` view, or
+  :guilabel:`Collections` screen, or
 
-- Click a collection in the left-hand pane.
+- Click a collection in the left navigation.
 
 .. figure:: /images/compass/select-collection.png
   :figwidth: 750px
 
-|compass| provides the following collection information in the
-detailed view:
+After you select a collection, |compass-short| shows you that
+collection's :doc:`/documents` tab. |compass-short| provides the following
```

Comment:
there's an extra "tab" here - the `/documents` link resolves to **Documents Tab** so this ends up as "**Documents Tab** tab"

In general I prefer to avoid this by always explicitly specifying the link text alongside the link, i.e.

```rst
:doc:`Documents Tab </documents>`
```

Reply:
oops ty for catching